invenio_cli/commands/containers.py

```diff
@@ -189,6 +216,29 @@ def static_pages(self, project_shortname):
 
         return steps
 
+    def translations(self, project_shortname):
+        """Steps to compile translations for the instance."""
+        commands = TranslationsCommands(
+            project_path=self.cli_config.get_project_dir(),
+            instance_path=self.cli_config.get_instance_path(),
+        )
+        cmd = commands.compile(
+            # instance path inside the container
+            directory="/opt/invenio/var/instance/translations",
```

Comment:
This whole function feels like an ugly hack to account for a containers environment. The instance path should not change? Can we maybe just pass as string `{INVENIO_INSTANCE_PATH}` and assume that is set from the docker image? (still not super beautiful...)

Reply:
@zzacharo Unfortunately no. The `get_instance_path` points to the local one stored in `.invenio.private` (e.g. `/Users/ppanero/.virtualenvs/instance-translations/var/instance`). In this case we need the instance path inside the container, namely https://github.com/inveniosoftware/docker-invenio/blob/master/almalinux/Dockerfile#L64.

Reply:
Can we maybe write this information in .invenio.private so it can be used for the containerized version? Still hardcoded but at least users can tweak that...wdyt?

Reply:
Discussed in real life: going with using the interpolation of `${INVENIO_INSTANCE_PATH}`